wordpress-mobile/MediaPicker-iOS
# Update project for Xcode 12 and iOS 14

This PR updates the code for Xcode 12 and iOS 14:

-  Check for PhotoLibrary permissions correctly in iOS 14.

On iOS 14 there is a new permission system for accessing the media library. 

The method to check for permission was deprecated and a new method was added: `requestAuthorizationForAccessLevel`

The answers to the new method have a new option called "PHAuthorizationStatusLimited" and it allows users to limit the media that is available to third-parties.

After a user selects this new option, the app will allow changing the media selected, after a "cold" restart.

This PR updates the code to use the new method when available. 

I'm considering if we should alert the user that they restricted their media access so only a small subset is available and direct the user to the settings to change that.

To test:
 - Start the demo app
 - Press the + button
 - Check that access to Photos permissions are requested and the option to "Select Media" works correctly.
